resources/ETradeOfferState.js

```diff
@@ -1,25 +1,36 @@
 module.exports = {
-	"Invalid": 1,
-	"Active": 2,            // This trade offer has been sent, neither party has acted on it yet.
-	"Accepted": 3,          // The trade offer was accepted by the recipient and items were exchanged.
-	"Countered": 4,         // The recipient made a counter offer
-	"Expired": 5,           // The trade offer was not accepted before the expiration date
-	"Canceled": 6,          // The sender cancelled the offer
-	"Declined": 7,          // The recipient declined the offer
-	"InvalidItems": 8,      // Some of the items in the offer are no longer available (indicated by the missing flag in the output)
-	"CreatedNeedsConfirmation": 9, // The offer hasn't been sent yet and is awaiting further confirmation
-	"CanceledBySecondFactor": 10, // Either party canceled the offer via email/mobile confirmation
-	"InEscrow": 11,          // The trade has been placed on hold
+	Invalid: 1,
+	// This trade offer has been sent, neither party has acted on it yet.
```

Comment:
I like having the documentation comment on the same line as what it's documenting. It isn't immediately obvious whether the comment is documenting the line above it, or the line below it.